### PR TITLE
fix(redshift): fix cache in connection wizard

### PR DESCRIPTION
**Pull request by yi-jin-0**
## Problem
1. When the connection parameters is removed, the re-launched connection wizard still show the earlier connection parameters.
2. When a sql query is successful but doesn't return any record, the output message is not clear. 
Screenshot 2023-10-20 at 11 52 12 AM
Screenshot 2023-10-20 at 11 51 17 AM

## Solution
1. Remove the connection wizard from the redshiftWarehouseNode constructor. Create a new instance when a connection wizard is launched.
2. Modify the output message when a sql query is successful but doesn't return any record.
Screenshot 2023-10-20 at 11 47 56 AM
Screenshot 2023-10-20 at 11 46 39 AM

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by sathiish-kumar — APPROVED** [review]

**Comment by justinmk3** [comment]
Does this have a customer-facing impact worth mentioning in the changelog?

To add a new changelog entry:

    npm run newChange

**Review by justinmk3 — APPROVED** [review]
Need a changelog

File: .changes/next-release/Bug Fix-7c49d65e-079c-4578-9c69-9dd7d4925b4e.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Redshift: 1. Remove the cache when the connection wizard is re-launched. 2. Modify the output message when a sql query is successful but doesn't return any record."
```
Comment (justinmk3) [inline .changes/next-release/Bug Fix-7c49d65e-079c-4578-9c69-9dd7d4925b4e.json:3]: - Is this description meaningful for customers? 
    - "Remove the cache" means nothing for anyone not familiar with the implementation details.
    - The _Problem_ described in the PR is closer to something that is actually meaningful. It describes the actual behavior that is being fixed.
- Use multiple `npm run newChange` commands to describe multiple changes. There is no need to put everything in one item.


```suggestion
	"description": "Redshift: re-launched connection wizard may use the old (stale) connection."
```

Comment (yi-jin-0) [reply .changes/next-release/Bug Fix-7c49d65e-079c-4578-9c69-9dd7d4925b4e.json]: Yes, I have made the suggested change.

**Review by justinmk3 — APPROVED** [review]

**Pull request merged**